Test/Invoke-WhiskeyExec.Tests.ps1

```diff
@@ -256,7 +258,7 @@ Describe 'Invoke-WhiskeyExec.when running an executable with multiple arguments'
 Describe 'Invoke-WhiskeyExec.when utilizing default task property to define executable and arguments' {

```

Comment:
This is what I'd like to see. Instead of passing the expected arguments to a `Given` function, pass them to an assertion.

```
Init
GivenExecutableFile 'executable.bat' 'exit 0'
GivenTaskDefaultProperty -Property 'executable.bat Arg1 Arg2 "Arg 3" ''Arg 4'''
WhenRunningExecutable
ThenExecutableRan 'executable.bat'
ThenSpecifiedArgumentsWerePassed 'Arg1','Arg2','"Arg 3"',"'Arg 4'"
```

Reply:
Updated tests to align with these suggestions.